src/core/hle/service/vi/vi.cpp

```diff
@@ -614,6 +614,14 @@ class IHOSBinderDriver final : public ServiceFramework<IHOSBinderDriver> {
             ctx.WriteBuffer(response.Serialize());
             break;
         }
+        case TransactionId::SetBufferCount: {
+            LOG_WARNING(Service_VI, "(STUBBED) called, transaction=SetBufferCount");
+            const auto buffer = ctx.ReadBuffer();
```

Comment:
You may wish to amend the commit (`git add -u`, then `git commit --amend`), with:

```suggestion
            [[maybe_unused]] const auto buffer = ctx.ReadBuffer();
```

Otherwise this can cause an unused variable warning.